### ISSUE TITLE
Is this interfacing the new g:profiler?

### ISSUE THREAD

**Issue by LuckyMD** [vals/python-gprofiler#8]
Hi!

A couple of months ago @liiskolb mentioned in a scanpy issue that g:profiler was getting/has a new interface. I was wondering if `python-gprofiler` interfaces with that or is still querying the old version?

Thanks!

**Comment by liiskolb** [comment]
@LuckyMD Based on the code it seems that this package is using the old version. 
We have a new API, etc. We have kept the archived version alive so everything works, but the annotation data (GO, KEGG, etc) is not up-to-date.

**Comment by LuckyMD** [comment]
@liiskolb Do you have a python API for the new g:profiler?

**Comment by liiskolb** [comment]
@LuckyMD Yes, we have updated gprofiler-official to version 1.0.0 that corresponds to the new API.
See the descriptions here: https://pypi.org/project/gprofiler-official/#description
